app/clients/sms/aws_sns.py

```diff
@@ -27,17 +38,32 @@ def send_sms(self, to, content, reference, multi=True, sender=None):
             matched = True
             to = phonenumbers.format_number(match.number, phonenumbers.PhoneNumberFormat.E164)
 
+            client = self._client
+            # See documentation
+            # https://docs.aws.amazon.com/sns/latest/dg/sms_publish-to-phone.html#sms_publish_sdk
+            attributes = {
+                'AWS.SNS.SMS.SMSType': {
+                    'DataType': 'String',
+                    'StringValue': 'Transactional'
+                }
+            }
+
+            # If sending with a long code number, we need to use another AWS region
+            # and specify the phone number we want to use as the origination number
+            send_with_dedicated_phone_number = self._send_with_dedicated_phone_number(sender)
+            if send_with_dedicated_phone_number:
+                client = self._long_codes_client
+                attributes['AWS.MM.SMS.OriginationNumber'] = {
+                    'DataType': 'String',
+                    'StringValue': sender
+                }
```

Comment:
If sending with a specific `sender`, switch the client to send texts from another AWS region and specify the sender as an attribute